<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Internet3GForm</class>
 <widget class="QWidget" name="Internet3GForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accès Internet Via Cia 3G</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QGroupBox" name="filialeGroup">
       <property name="title">
        <string>Filiale</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="cle3g_filiale_lineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="dateGroup">
       <property name="title">
        <string>Date</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QDateEdit" name="cle3g_date_dateEdit">
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="cle3g_renouvellement_radio">
     <property name="text">
      <string>renouvellement</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="cle3g_nouvelleDemande_radio">
     <property name="text">
      <string>nouvelle demande</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Demande d'accès à Internet Via Cia 3G</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_nom">
       <property name="text">
        <string>Nom et Prénom :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="cle3g_nomPrenom_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_service">
       <property name="text">
        <string>Service :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="cle3g_service_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_anciennete">
       <property name="text">
        <string>Ancienneté :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cle3g_anciennete_lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="motifsGroup">
     <property name="title">
      <string>Motifs, impacts escomptés :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="cle3g_motif_textEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="conditionsGroup">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="conditionLabel1">
        <property name="text">
         <string>- L'accès à Internet est destiné uniquement à des utilisations professionnelles en relation directe avec la nature des fonctions du demandeur. Le mot de passe ne doit pas être communiqué à d'autres personnes.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="conditionLabel2">
        <property name="text">
         <string>- Il est strictement interdit de télécharger des utilitaires ou programmes ou fichiers non professionnels.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="declarationLabel">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Je déclare avoir pris connaissance que tout abus lors de mes accès à internet entraînerais la suspension immédiate et automatique de ma connexion.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cle3g_submit_pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Soumettre la demande</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
